README.md

```diff
@@ -30,111 +30,259 @@ Recommended Setup
 
 Before you can start, you have to download the browscap.ini file and convert it into a cache. There are two ways.
 
-1. Download the file and convert it in two steps. The downloaded file will be stored in a local file, but there is no check
+a. Download the file and convert it in two steps. The downloaded file will be stored in a local file, but there is no check
    if the remote file has changed. If your cache gets corrupted you only need to rerun the `convert` command.
 
-```php
-vendor/bin/browscap-php browscap:fetch
-vendor/bin/browscap-php browscap:convert
-```
+    ```php
```

Comment:
You can use either 4-spaces to indent code, or 

    ```php
    // your code here
    ```

The latter is preferable please :)